string task
string marker
---
string result